groth16/Cargo.toml

```diff
@@ -0,0 +1,48 @@
+[package]
+name = "groth16"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+byteorder = "1"
+franklin-crypto = { git = "https://github.com/matter-labs/franklin-crypto", branch = "beta", features = [ "plonk" ], version = "0.0.5"}
```

Comment:
We don't have to import franklin_crypto, as its an implement of `plonk`. 

For here, we  can use the `bellman_ce`, `pairing` crate instead `franklin`.

Reply:
Fixed by cargo udeps

Reply:
Another thought is we have a GPU frankin crypto. Can you plz create an issue to discuess this?

Reply:
Sure. see #129

Reply:
it supports, but we have not done the integration test for a while.